domains/blatant.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "jjjjjjjns",
+           "email": "63760073+jjjjjjjns@users.noreply.github.com",
+           "discord": "749473211594309714"
+        },
+    
+        "record": {
+            "TXT": "shrekmandev"
```

Comment:
For github pages you should use CNAME records

Reply:
ok